clouddriver-web/config/clouddriver.yml

```diff
@@ -248,6 +248,11 @@ cf:
       username: ${cf.account.name}
       password: ${cf.account.password}
 
+spring:
+  jackson:
+    serialization:
+      ORDER_MAP_ENTRIES_BY_KEYS: true
```

Comment:
Done! Nice find - I removed all the annotations as this does the same thing in one place.